src/components/app/details/triggerView/workflow.service.ts

```diff
@@ -57,6 +58,36 @@ export function processWorkflow(
     let ciPipelineToNodeWithDimension = (ciPipeline: CiPipeline) => ciPipelineToNode(ciPipeline, dimensions)
     const filteredCIPipelines =
         ciResponse?.ciPipelines?.filter((pipeline) => pipeline.active && !pipeline.deleted) ?? []
+    const configuredMaterialList = new Map()
+    for (const ciPipeline of filteredCIPipelines) {
+        configuredMaterialList[ciPipeline.name] = new Set()
+        if (ciPipeline?.ciMaterial?.length > 0) {
```

Comment:
remove optional checks for ciPipeline as if filteredCIPipelines is empty then it'll not reach here

Reply:
done